xtask/src/xtask.rs

```diff
@@ -182,6 +182,13 @@ fn test_tmt(sh: &Shell) -> Result<()> {
 
     cmd!(sh, "cargo run -p tests-integration run-vm prepare-tmt").run()?;
 
+    // pull some small images that are used for LBI installation tests
+    cmd!(
+        sh,
+        "podman pull quay.io/curl/curl-base:latest quay.io/curl/curl:latest"
```

Comment:
Minor: probably worth factoring out a `const TEST_IMAGES: &[&str] = ...` at the top of the file for increased visibility?

This one actually also relates to https://gitlab.com/fedora/bootc/examples/-/merge_requests/50#note_2041291332
which hmmm, I need to finish that PR